video/csputils.h

```diff
@@ -121,7 +121,8 @@ struct mp_colorspace {
     enum mp_csp_levels levels;
     enum mp_csp_prim primaries;
     enum mp_csp_trc gamma;
-    float peak; // 0 = auto/unknown
+    float nom_peak; // nominal (absolute) peak. 0 = auto/unknown
+    float ref_peak; // reference (white) peak
```

Comment:
Well, pretty much everything related to color metadata etc. in mpv is float already. It's just more or less the status quo at this point.

(I've been thinking about changing everything to double instead, which might matter for stuff like the CMS matrix, but the precision is negligible for the peak - which only matters to within 100 cd/m² or so)